os/src/Path.scala

```diff
@@ -481,7 +481,8 @@ sealed trait PathConvertible[T]{
 
 object PathConvertible{
   implicit object StringConvertible extends PathConvertible[String]{
-    def apply(t: String) = java.nio.file.Paths.get(t)
+    def apply(t: String) =
+      java.nio.file.Paths.get(if(scala.util.Properties.isWin && t.matches("\\/[A-z]:\\/.*")) t.drop(1) else t)
```

Comment:
```suggestion
      java.nio.file.Paths.get(if(scala.util.Properties.isWin && t.matches("\\/[A-Za-z]:\\/.*")) t.drop(1) else t)
```

As @sake92  commented earlier, to avoid matching some non-alphabetic character.